Makefile

```diff
@@ -157,6 +168,23 @@ deploy-test: token-check k8s-env clean-deployment deploy-targets proxy-test
 
 #Testing code, configuration and deployment changes
 integration-test: token-check k8s-env clean-deployment deploy-targets delete-images push-images proxy-test
+	echo "Integration test finished. To generate code coverage reports, run 'make coverage-test'"
```

Comment:
I'd suggest removing this comment, it will become a bit redundant after a while

Reply:
Will do